src/components/Motorcycle/AddForm/index.js

```diff
@@ -0,0 +1,320 @@
+                    />
+                  </FloatingLabel>
+                  <Button
+                    variant="outline-info fw-bolder"
+                    style={{ fontFamily: "'Rubik', sans-serif" }}
+                    class="btn-lg p-3"
+                    onClick={() => (state.image
+                      ? !(state.images.find(({ image }) => image === state.image))
+                        ? setState((s) => ({
+                          ...s,
+                          images: [...s.images, {
+                            id: generate({ charset: 'number' }),
+                            image: s.image,
+                          }],
+                          image: '',
+                        }))
+                        : showErrorToast('Please add another image link', addToast)
+                      : showErrorToast('Please add image link', addToast))}
+                  >
+                    + Add image
+                  </Button>
+                </div>
+                <div className="row w-100 mt-4">
+                  <Carousel
+                    keyBoardControl
+                    className="w-100"
+                    itemClass="mr-10"
+                    responsive={responsive([3, 2, 1])}
+                  >
+                    {
+                    state.images.map(({ image, id }) => (
+                      <div
+                        className="col-lg-12 w-100"
+                        key={id}
+                        style={{ marginRight: 10 }}
+                      >
+                        <div className="card w-100">
+                          <img
+                            src={image}
+                            alt="Snow"
+                            className="w-100"
+                            style={{ height: '200px', objectFit: 'cover' }}
+                          />
+                          <Button
+                            type="button"
+                            variant="outline-danger"
+                            className="fa-2x fw-bolder"
+                            style={{ fontFamily: "'Rubik', sans-serif" }}
+                            onClick={() => removeImage(id)}
+                          >
+                            Remove
+                          </Button>
+                        </div>
+                      </div>
+                    ))
+                 }
+                  </Carousel>
+                </div>
+
+                <Button
+                  variant="outline-success"
+                  type="submit"
+                  className="w-100 mt-4 fw-bolder"
+                  style={{ fontFamily: "'Rubik', sans-serif" }}
+                >
+                  {params?.id
+                    ? 'Edit Motorcycle' : '+ Add Motorcycle'}
+
+                </Button>
+              </div>
+            </form>
+          </div>
+          <div className="col-lg-3">{' '}</div>
+        </div>
+      </div>
+    </AllContainer>
```

Comment:
This returned component should definitely be simplified. It's very long, maybe it can be a component in its own right?
Also:
There are some candidates to make some of these html components reusable.